### PR TITLE
[FR] Add `Skinning\History`

### PR DESCRIPTION
**Pull request by Ganondorf-1**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Skinning/History/fr.md [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+
+## Interface
+
+`menu-osu.png`
+
+![](img/menu-osu.png)
+
+| Versions | Peut être animé | Skinnable sur la beatmap | Mode de fusion | Origine | Taille SD suggérée |
+| :-: | :-: | :-: | :-: | :-: | :-: |
+| Toutes | ![No][false] | ![Yes][true] | Normal | Centre | - |
```
Comment (Walavouchey) [inline wiki/Skinning/History/fr.md:13]: etc

```suggestion
| Toutes | ![Non][false] | ![Oui][true] | Normal | Centre | - |
```

File: wiki/Skinning/History/fr.md [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+
+---
+
+`fps-3.png`
+
+![](img/fps-3.png)
+
+| Versions | Peut être animé | Skinnable sur la beatmap | Mode de fusion | Origine | Taille SD suggérée |
+| :-: | :-: | :-: | :-: | :-: | :-: |
+| Toutes | ![Non][false] | ![Oui][true] | Normal | (inconnu) | - |
```
Comment (Biokax) [inline wiki/Skinning/History/fr.md:176]:
```suggestion
| Toutes | ![Non][false] | ![Oui][true] | Normal | (inconnue) | - |
```

File: wiki/Skinning/History/fr.md [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+
+---
+
+`fps-5.png`
+
+![](img/fps-5.png)
+
+| Versions | Peut être animé | Skinnable sur la beatmap | Mode de fusion | Origine | Taille SD suggérée |
+| :-: | :-: | :-: | :-: | :-: | :-: |
+| Toutes | ![Non][false] | ![Oui][true] | Normal | (inconnu) | - |
```
Comment (Biokax) [inline wiki/Skinning/History/fr.md:208]:
```suggestion
| Toutes | ![Non][false] | ![Oui][true] | Normal | (inconnue) | - |
```

File: wiki/Skinning/History/fr.md [hunk 4]
```diff
@@ -0,0 +1,350 @@
...
+
+---
+
+`fps-8.png`
+
+![](img/fps-8.png)
+
+| Versions | Peut être animé | Skinnable sur la beatmap | Mode de fusion | Origine | Taille SD suggérée |
+| :-: | :-: | :-: | :-: | :-: | :-: |
+| Toutes | ![Non][false] | ![Oui][true] | Normal | (inconnu) | - |
```
Comment (Biokax) [inline wiki/Skinning/History/fr.md:256]:
```suggestion
| Toutes | ![Non][false] | ![Oui][true] | Normal | (inconnue) | - |
```

File: wiki/Skinning/History/fr.md [hunk 5]
```diff
@@ -0,0 +1,350 @@
...
+
+---
+
+`fps-comma.png`
+
+![](img/fps-comma.png)
+
+| Versions | Peut être animé | Skinnable sur la beatmap | Mode de fusion | Origine | Taille SD suggérée |
+| :-: | :-: | :-: | :-: | :-: | :-: |
+| Toutes | ![Non][false] | ![Oui][true] | Normal | (inconnu) | - |
```
Comment (Biokax) [inline wiki/Skinning/History/fr.md:288]:
```suggestion
| Toutes | ![Non][false] | ![Oui][true] | Normal | (inconnue) | - |
```

File: wiki/Skinning/History/fr.md [hunk 6]
```diff
@@ -0,0 +1,350 @@
...
+
+---
+
+`fps-fps.png`
+
+![](img/fps-fps.png)
+
+| Versions | Peut être animé | Skinnable sur la beatmap | Mode de fusion | Origine | Taille SD suggérée |
+| :-: | :-: | :-: | :-: | :-: | :-: |
+| Toutes | ![Non][false] | ![Oui][true] | Normal | (inconnu) | - |
```
Comment (Biokax) [inline wiki/Skinning/History/fr.md:304]:
```suggestion
| Toutes | ![Non][false] | ![Oui][true] | Normal | (inconnue) | - |
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**